concierge/service/src/main/java/org/eclipse/ditto/concierge/service/enforcement/PolicyCommandEnforcement.java

```diff
@@ -97,16 +104,27 @@ private PolicyCommandEnforcement(final Contextual<PolicyCommand<?>> context, fin
      * @param command the command to authorize.
      * @return optionally the authorized command.
      */
-    public static <T extends PolicyCommand<?>> Optional<T> authorizePolicyCommand(final T command,
-            final PolicyEnforcer policyEnforcer) {
+    public static <T extends PolicyCommand<?>> Optional<T> authorizePolicyCommand(final Contextual<?> context,
```

Comment:
```suggestion
    public static <T extends PolicyCommand<?>> Optional<T> authorizePolicyCommand(
```
Please remove the added `context` param again, as it is not used.